consensus/poet/core/sawtooth_poet/poet_consensus/poet_settings_view.py

```diff
@@ -16,12 +16,12 @@
 import math
```

Comment:
I thought that our normal approach to renaming that also caused changes to the file was to split this into two commits - one commit was simply renaming the files.  The second commit made any fix-ups to the file.  My understanding was even though it meant that the first commit pulled in isolation would result in a broken system, it was cleaner and clearer to separate out the renaming from the updates.